Source/Core/VideoCommon/ShaderCache.cpp

```diff
@@ -445,7 +446,11 @@ ShaderCache::CompilePixelUberShader(const UberShader::PixelShaderUid& uid) const
 {
   const ShaderCode source_code =
       UberShader::GenPixelShader(m_api_type, m_host_config, uid.GetUidData());
-  return g_renderer->CreateShaderFromSource(ShaderStage::Pixel, source_code.GetBuffer());
+  const UberShader::pixel_ubershader_uid_data* data = uid.GetUidData();
+  std::string name = fmt::format("Pixel UberShader for {} texgens{}{}", data->num_texgens,
+                                 data->early_depth ? ", early depth" : "",
+                                 data->per_pixel_depth ? ", per-pixel depth" : "");
+  return g_renderer->CreateShaderFromSource(ShaderStage::Pixel, source_code.GetBuffer(), name);
```

Comment:
Because I stole the code from the ubershader header generation and it didn't either

I moved the name to a custom `fmt` stringify, if that makes more sense

Reply:
Yep, that seems like a good idea to me.